pages/api/ipfs/index.ts

```diff
@@ -0,0 +1,137 @@
+import { IOMarketMetadata } from "@zeitgeistpm/sdk";
+import type { PageConfig } from "next";
+import { NextRequest } from "next/server";
+
+const CLUSTER_ENDPOINT = process.env.IPFS_CLUSTER_URL;
+
+const headers = {
+  Authorization: `Basic ${Buffer.from(
+    `${process.env.IPFS_CLUSTER_USERNAME}:${process.env.IPFS_CLUSTER_PASSWORD}`,
```

Comment:
Those will never be exposed to the end user?

Reply:
No those are only loaded on the server. Also vercel/next prevents any env varible that isnt prefixed with `NEXT_PUBLIC` from being sent to the client.